### PR TITLE
Fixes #27577 - Fail test on console.error

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/common/Slot/Slot.test.js [hunk 1]
```diff
@@ -160,6 +167,7 @@ describe('Slot-Fill', () => {
   });
 
   it('multi slot with override props fill amd component fill', () => {
```
Comment (ekohl) [inline webpack/assets/javascripts/react_app/components/common/Slot/Slot.test.js]: Unrelated typo:

```suggestion
  it('multi slot with override props fill and component fill', () => {
```

**Review by sharvit — COMMENTED** [review]
Thanks @xprazak2 LGTM, please fix the linting errors.

**Comment by xprazak2** [comment]
I fixed the linting.

**Review by sharvit — DISMISSED** [review]

**Review by ekohl — DISMISSED** [review]
These are the things I like: proper CI testing and catching errors. Going trust @sharvit's review of the actual changes.

**Comment by ekohl** [comment]
[test katello]

**Comment by sharvit** [comment]
Should we rebase and restest before the merge?

**Comment by sharvit** [comment]
> Should we rebase and restest before the merge?

Just because a new code might cause it to fail?

**Comment by ekohl** [comment]
I would like that since we've merged some JS changes since.

**Comment by ekohl** [comment]
@xprazak2 could you rebase to make sure we don't introduce regressions?

**Comment by xprazak2** [comment]
Rebased.

**Review by sharvit — APPROVED** [review]

**Review by ezr-ondrej — APPROVED** [review]
Thanks @xprazak2, thanks @sharvit for the review :+1:
Merging! (sorry to steal this one @ekohl)

**Pull request merged**